rest/functionsapitest/main_test.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2020-Present Couchbase, Inc.
+
+Use of this software is governed by the Business Source License included in
+the file licenses/BSL-Couchbase.txt.  As of the Change Date specified in that
+file, in accordance with the Business Source License, use of this software will
+be governed by the Apache License, Version 2.0, included in the file
+licenses/APL2.txt.
+*/
+
+package functionsapitest
+
+import (
+	"testing"
+
+	"github.com/couchbase/sync_gateway/db"
+)
+
+func TestMain(m *testing.M) {
+	memWatermarkThresholdMB := uint64(18192)
```

Comment:
Huh, I'm not sure because I didn't write that -- it was one of the new hires in India. It sure could be a typo. 

I don't even know what that parameter controls; is it a memory limit?